insights/tests/parsers/test_uname.py

```diff
@@ -236,19 +239,42 @@ def test_fixed_by():
     assert ['2.6.33-100.el6'] == u.fixed_by('2.6.33-100.el6')
     assert ['2.6.32-600.el6'] == u.fixed_by('2.6.32-220.1.el6', '2.6.32-600.el6')
 
+    # For all remaining tests, cause the warnings to always be caught
```

Comment:
Better add a new test function to wrap the following tests for the warnings.

And better call `warnings.resetwarnings()` to reset the warning filters after the test. (at the end of a new added tet function.

Reply:
Thanks, updated.

Reply:
@xiangce thanks, updated.